.golangci-kal.yml

```diff
@@ -58,33 +58,21 @@ issues:
   max-issues-per-linter: 0
   exclude-rules:
   # KAL should only run on API folders.
-  - path-except: "api/*"
-    linters:
-      - kal
-  # apiwarnings contains a `api` characters in the path, but it's not an API folder.
-  - path: "util/apiwarnings/*"
+  - path-except: "api//*"
     linters:
       - kal
-  - path: "api/v1beta1/*|api/v1alpha1/*"
+  - path: "api//*"
     text: "Conditions field must be a slice of metav1.Condition"
     linters:
       - kal
+  # We follow the current CustomResourceDefinition field's json tag pattern.
   - path: "api/v1beta1/*"
-    text: "field XPreserveUnknownFields json tag does not match pattern"
-    linters:
-      - kal
-  - path: "api/v1beta1/*"
-    text: "field XValidations json tag does not match pattern"
-    linters:
-      - kal
-  - path: "api/v1beta1/*"
-    text: "field XMetadata json tag does not match pattern"
-    linters:
-      - kal
-  - path: "api/v1beta1/*"
-    text: "field XIntOrString json tag does not match pattern"
+    source: "X.*"
```

Comment:
I think this is too broad, can we be specific about the field names we are excepting here please

Reply:
Absolutely. We have something like this in our main .golangci.yml. This here should be similar

```yaml
    text: "SA1019: (bootstrapv1.ClusterStatus|KubeadmConfigSpec.UseExperimentalRetryJoin|scope.Config.Spec.UseExperimentalRetryJoin|DockerMachine.Spec.Bootstrapped|machineStatus.Bootstrapped|dockerMachine.Spec.Backend.Docker.Bootstrapped|devMachine.Spec.Backend.Docker.Bootstrapped|c.TopologyPlan|clusterv1.ClusterClassVariableMetadata|(variable|currentDefinition|specVar|newVariableDefinition|statusVarDefinition).Metadata) is deprecated"
```

Reply:
Exactly. fixed.